### PR TITLE
rename glimmershow

### PR DESCRIPTION
**Pull request by MilonPL**
## About the PR

Changed "glimmershow" to "showglimmer"

## Why / Balance

All of the commands start with "show". OCD.

## Media
- [X] This PR does not require an ingame showcase

**Changelog**

:cl:
DELTAVADMIN:
- tweak: Renamed "glimmershow" to "showglimmer".

### CODE REVIEW

File: Content.Server/Nyanotrasen/Psionics/Glimmer/GlimmerCommands.cs [hunk 1]
```diff
@@ -5,12 +5,12 @@
 
 namespace Content.Server.Psionics.Glimmer;
 
-[AdminCommand(AdminFlags.Logs)]
-public sealed class GlimmerShowCommand : IConsoleCommand
+[AdminCommand(AdminFlags.Debug)] // Delta-V - Set to Debug flag
```
Comment (Axionyxx) [inline Content.Server/Nyanotrasen/Psionics/Glimmer/GlimmerCommands.cs]: I know this'd make it match the set command, but given all it does is show the glimmer, just admin would be fine

**Review by Axionyxx — CHANGES_REQUESTED** [review]

**Comment by deltanedas** [comment]
glimmerset: :trollface:

also i dont think the comments matter for nyano files since le dead

**Comment by MilonPL** [comment]
> also i dont think the comments matter for nyano files since le dead

i can't risk a lawsuit in this economy

**Comment by deltanedas** [comment]
well to not risk lawsuit youd have to keep the mit license notice in every nyano file, not just add `# stolen from nyano` everywhere :trollface:

**Review by Axionyxx — APPROVED** [review]

**Pull request merged**